### PR TITLE
Updated queries to work with the website's new API requests. User can choose number of cards. Saves to text file.

### PR DESCRIPTION
**Pull request by ariabee**
sorry, I changed 'main.py' to 'tabooScraper.py' and now the updates aren't noticeable

### CODE REVIEW

File: tabooScraper.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    return numCards
+
+
+numCards = promptUser()
+print("Printing",numCards,"cards:")
+
+# Print each taboo card and save output to file.
+with open("taboo.txt", "a+") as file:
+    for num in range(numCards):
+        url = "https://playtaboo.com/ajax/v1/next/?" + str(num)
```
Comment (rohitnawani) [inline tabooScraper.py]: can you pull this out to the top as constant? something like..
`TABOO_URL = "https://playtaboo.com/ajax/v1/next/?"`

File: tabooScraper.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        print("That wasn't a number, try again?")
+        numCards = promptUser()
+    return numCards
+
+
+numCards = promptUser()
+print("Printing",numCards,"cards:")
+
+# Print each taboo card and save output to file.
+with open("taboo.txt", "a+") as file:
```
Comment (rohitnawani) [inline tabooScraper.py]: same here with `taboo.txt`

File: tabooScraper.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+def parseOutput(url):
+    '''
+    Examples Input/Output
+    Input: https://playtaboo.com/ajax/v1/next/?1624643694954
+    Output: (GOVERNOR, ['STATE', 'REPRESENTATIVE', 'LEADER', 'EACH', 'ELECT'])
+    '''
+    string = str(urlopen(url).read())
+
+    gameWordQuery = r'game-word\\\\">'+'(.*?)'+r'<\\\\/h2>'
```
Comment (rohitnawani) [inline tabooScraper.py]: these new queries seem complicated, not sure if there is an easier to achieve the same

Comment (rohitnawani) [reply tabooScraper.py]: but it works 👍🏾

File: tabooScraper.py (hunk 1 above)
Comment (ariabee) [reply tabooScraper.py]: yes, great idea. I'll do that next!

File: tabooScraper.py (hunk 2 above)
Comment (ariabee) [reply tabooScraper.py]: yes yes

**Comment by rohitnawani** [comment]
@ariabee - thanks for fixing the URL. 🥇  
I personally prefer `main.py` because I like to just run `python main.py` in all python projects 🐰 but it's your call. 
Tested, LGTM, will approve once you address the minor chore items

File: tabooScraper.py (hunk 3 above)
Comment (ariabee) [reply tabooScraper.py]: so, `r'...'` is the raw text found in the string of the web page. In other words, the query is really `r'____' + '(.*?)' + r'___'`. But I could add a comment to make this info clear

**Comment by ariabee** [comment]
@rohitnawani awesome! Okay, I can change it back to main.py 🐰 since it's in the tabooWebScraper folder anyway. Time for chores

**Comment by ariabee** [comment]
bunny-result
:)

**Pull request merged**